installers/setupEvents.js

```diff
@@ -0,0 +1,67 @@
+const electron = require('electron')
+const app = electron.app
+
+module.exports = {
```

Comment:
This file needs documentation; what is it used for? It looks like it handles the windows installer.